pact/verify_wrapper.py

```diff
@@ -119,12 +150,26 @@ class PactException(Exception):
 
     def __init__(self, *args, **kwargs):
         """Create wrapper."""
+        warnings.warn(
+            "This class will be deprecated Pact Python v3 "
+            "(see pact-foundation/pact-python#396)",
+            PendingDeprecationWarning,
+            stacklevel=2,
+        )
         super().__init__(*args, **kwargs)
         self.message = args[0]
 
 class VerifyWrapper(object):
     """A Pact Verifier Wrapper."""
 
+    def __init__(self):
+        warnings.warn(
+            "This class will be deprecated Pact Python v3 "
+            "(see pact-foundation/pact-python#396)",
```

Comment:
Not sure whether that's really necessary since GitHub shares the same numbering sequence across both PRs and issues and GitHub automatically redirects to the appropriate one:

- [`https://github.com/pact-foundation/pact-python/issues/396`](https://github.com/pact-foundation/pact-python/issues/396)
- [`https://github.com/pact-foundation/pact-python/pull/396`](https://github.com/pact-foundation/pact-python/pull/396)

In hindsight, I probably should have put the message in a const (with a placeholder for `class`/`function`)...